internal/pkg/print/print.go

```diff
@@ -18,6 +21,8 @@ const (
 	InfoLevel    Level = "info"
 	WarningLevel Level = "warning"
 	ErrorLevel   Level = "error"
+
+	NoneOutputFormat = "none"
```

Comment:
This const seems to belong together with `JSONOutputFormat` and `PrettyOutputFormat`, but we can't import from there in this file

Reply:
But maybe it makes even more sense if the 3 of them are defined here in the `print` package